eclair-node/src/main/scala/fr/acinq/eclair/Boot.scala

```diff
@@ -48,8 +48,8 @@ object Boot extends App with Logging {
 
   def onError(t: Throwable): Unit = {
     val errorMsg = if (t.getMessage != null) t.getMessage else t.getClass.getSimpleName
-    System.err.println(s"fatal error: $errorMsg")
-    logger.error(s"fatal error: $errorMsg")
+    logger.error(s"fatal error: $errorMsg", t)
+    t.printStackTrace()
```

Comment:
`errorMsg` should be removed because the message is already printed with the stack trace